Classes/Other/Secrets.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 public enum GithubAPI {
 
-    static let clientID = "YOUR_CLIENT_ID"
-    static let clientSecret = "YOUR_CLIENT_SECRET"
+    static let clientID = "d611a03290c738ba4f3f"
```

Comment:
Can you remove your GitHub API details please! (We really should add this to the `.gitignore` to stop people making this mistake 😰)

Reply:
oooh 😰😰😰

Remove from Github or Project? 

Sorry about that! 
But yes, we should add it to .gitignore

Reply:
Just from the project, leaving it unchanged from where it was -- Up to you if you want to regenerate your Oauth app to receive new ID/secrets just in case.

Reply:
It should be changed back!